tasks/otel_agent.py

```diff
@@ -69,3 +69,15 @@ def image_build(ctx, arch='amd64', base_version='latest', tag=OT_AGENT_TAG, push
 
     os.remove(os.path.join(build_context, BIN_NAME))
     os.remove(os.path.join(build_context, CFG_NAME))
+
+
+@task
+def integration_test(ctx):
+    """
+    Run the otel integration test
+    """
+    cmd = """go test -timeout 0s -tags kubelet,jmx,cri,otlp,zstd,zk,jetson,etcd,\
```

Comment:
are you sure we need all these tags?  don't we just need -otlp and maybe -test?

Reply:
probably fine to do that instead